src/DeepHelloWorld/Tests/Backend/test/SayHello/CreateDb/functional.spec.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+import chai from 'chai';
+import dir from 'node-dir';
+import path from 'path';
+import {Exec} from '../../../node_modules/deepify/lib.compiled/Helpers/Exec';
+
+let expect = chai.expect;
+
+suite('Functional tests', () => {
+
+  let inputEventsArray = [];
+  let inputEventsFilesArray = [];
+  let expectedResultsArray = [];
+  let expectedResultsFilesArray = [];
+  let i = 0;
+
+  suiteSetup((done) => {
+
+    const TEST_ASSERTS_DIR = '../../../../test/SayHello/CreateDb/test-asserts';
+    let dirPath = path.join(__dirname, TEST_ASSERTS_DIR);
+
+    dir.readFiles(dirPath, {
+        match: /result.json$/,
+        exclude: /^\./,
+      }, (err, content, next) => {
+        if (err) throw err;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'throw'.](https://www.codacy.com/app/MitocGroup/deep-microservices-helloworld/file/2506095382/issues/source?bid=2928544&fileBranchId=3275273#l27)